curvefs/src/client/common/config.cpp

```diff
@@ -322,8 +331,8 @@ void SetFuseClientS3Option(FuseClientOption *clientOption,
     clientOption->s3Opt.s3AdaptrOpt.bucketName = fsS3Opt.bucketName;
 }
 
-void S3Info2FsS3Option(const curvefs::common::S3Info& s3,
-                       S3InfoOption* fsS3Opt) {
+void S3Info2FsS3Option(const curvefs::common::S3Info &s3,
+                       S3InfoOption *fsS3Opt) {
```

Comment:
these changes look like only about code styles, the old and new styles in C++ are all good, we can leave them to old style, so the reviewer can pay more attentions to your added codes.